packages/react-components/source/react/library/icon/icons.js

```diff
@@ -579,6 +579,16 @@ const icons = {
     ),
   },
 
+  'empty-circle': {
+    medium: (
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M8 0C12.411 0 16 3.589 16 8C16 12.411 12.411 16 8 16C3.589 16 0 12.411 0 8C0 3.589 3.589 0 8 0ZM8 14C11.309 14 14 11.309 14 8C14 4.691 11.309 2 8 2C4.691 2 2 4.691 2 8C2 11.309 4.691 14 8 14Z"
+      />
```

Comment:
This may not have been run through svgo? Not a big deal for a small icon, but I like following that [process](https://github.com/puppetlabs/design-system/blob/main/packages/react-components/source/react/library/icon/Icon.md#adding-icons-to-the-design-system) since we might as well save space when we can.

Just ran it; here, see if this works:

```suggestion
      <path d="M8 0c4.411 0 8 3.589 8 8s-3.589 8-8 8-8-3.589-8-8 3.589-8 8-8Zm0 14c3.309 0 6-2.691 6-6s-2.691-6-6-6-6 2.691-6 6 2.691 6 6 6Z" />
```

Reply:
Yep, I missed that step. I'll update this.